clients/admin-ui/src/types/api/models/ClassifyParams.ts

```diff
@@ -8,7 +8,7 @@
  * and can rely on the defaults
  */
 export type ClassifyParams = {
-  possible_targets?: Array<any>;
+  possible_targets?: null;
```

Comment:
why does this return only null?

Reply:
Hmm, ok, I'll update back to using `Array<any>` for now.

Reply:
@eastandwestwind did I accidentally override this? Should I put back to Array<any>?

Reply:
If the tests pass, then this is fine as `null`. I don't want to run into issues in the future where every time we autogenerate types, we have to do manual changes like this.

I think the real question is how we get pydantic to better transform types like `Optional[Iterable]` from the BE.

Reply:
Thank you 🙏